.github/renovate-config.json5

```diff
@@ -5,7 +5,7 @@
   platform: "github",
   repositories: ["jenkinsci/helm-charts"],
   // eslint-disable-next-line
-  allowedPostUpgradeCommands: ['^\.github\/renovate-postupgrade\.sh "{{{depName}}}" "{{{newVersion}}}"$'],
+  allowedPostUpgradeCommands: ['^\.github\/renovate-postupgrade\.sh ".+" ".+"$'],
```

Comment:
@kvanzuijlen what do you think?

I don't really want to list all deps here, could restrict their characters for the first one second is also do-able but more of a pain to get all characters